packages/neutrino-preset-jest/src/index.js

```diff
@@ -92,13 +93,26 @@ module.exports = (neutrino, opts = {}) => {
     });
 
     return new Promise((resolve, reject) => {
+      // We need to parse argv separately in order to identify files
+      // and jest-related options since root neutrino does not know about
+      // jest options and will provide wrong/incomplete `args.files`
+      const jestArgs = yargs
+        .command('test [files..]', 'Run tests', jestOptions)
+        .argv;
       const configFile = join(tmpdir(), 'config.json');
       const options = normalizeJestOptions(opts, neutrino, args);
```

Comment:
Since `normalizeJestOptions` no longer accepts `args`, we can remove them from being passed here.

Reply:
Good catch! Completely forgot about it here!

Reply:
Done!